.github/workflows/promptflow-replay-test.yml

```diff
@@ -0,0 +1,105 @@
+name: promptflow-replay-test
+on:
+  pull_request:
+    paths:
+      - src/promptflow/**
+      - scripts/**
+  workflow_dispatch:
+env:
+  testWorkingDirectory: ${{ github.workspace }}/src/promptflow
+  PYTHONPATH: ${{ github.workspace }}/src/promptflow
+  IS_IN_CI_PIPELINE: "true"
+  PF_RECORDING_MODE: "replay"
```

Comment:
shall you and @zhengfeiwang us same environment variable? 
I think zhengfei is using something like:
PROMPT_FLOW_TEST_RUN_LIVE

Reply:
OKay. let's do this.